eng/testing/.runsettings

```diff
@@ -32,7 +32,7 @@
   </LoggerRunSettings>
   <DataCollectionRunSettings>
     <DataCollectors>
-      <DataCollector friendlyName="XPlat code coverage" enabled="$$COVERAGE_ENABLED$$">
+      <DataCollector friendlyName="XPlat code coverage" enabled="false">
```

Comment:
What does this change mean

Reply:
This means that code coverage isn't collected by default. The section here is only to configure the coverlet data collector. As officially documented by coverlet (https://github.com/coverlet-coverage/coverlet#usage), you need to specify the data collector on the CLI `--collect:"XPlat Code Coverage"`. I know that this isn't ideal as devs are used to just specify `/p:Coverage=true` but with VSTest there is currently no other way to enable both the in- and out-of-proc collector.

I started working on making that possible in https://github.com/microsoft/vstest/pull/2429 a few months ago.